llvm/lib/Transforms/InstCombine/InstructionCombining.cpp

```diff
@@ -5099,6 +5099,49 @@ Instruction* cs6475_optimizer(Instruction *I) {
   }
   // END JOHN REGEHR
 
+  // BEGIN SINA MAHDIPOUR SARAVANI
```

Comment:
please put your code inside of a {  } block so that your local variables do not escape-- lots of people will be using the same variable names